### DISCUSSION TITLE
carousel走马灯 无法动态绑定是否autoplay

### DISCUSSION THREAD

**Discussion by Zwei-J** [vueComponent/ant-design-vue#3533] [Q&A]
https://codesandbox.io/s/vue-antd-template-forked-15idu?file=/src/App.vue

**Comment by Zwei-J** [comment]
@TaoHuaXia

**Comment by ajuner** [comment]
代码的底层逻辑应该是没有问题的，单组件引入跟全局引入Antd分别试试

**Comment by sendya** [comment]
可以暂时 v-if 重新渲染一下
